models/sessions/scratch/default/snowplow_web_sessions_this_run.sql

```diff
@@ -302,6 +366,11 @@ select
         {%- for conv_def in var('snowplow__conversion_events') %}
     {{ snowplow_web.get_conversion_columns(conv_def, names_only = true)}}
         {%- endfor %}
+    {% if var('snowplow__total_all_conversions', false) %}
```

Comment:
Should we include `var('snowplow__conversion_events')` in the `if` statement here?

Reply:
It's already nested within that if block